.appveyor.yml

```diff
@@ -4,9 +4,9 @@ environment:
 
 install:
   - ps: Install-Product node $env:nodejs_version
-  - ps: (new-object net.webclient).DownloadFile('https://github.com/mbuilov/gnumake-windows/raw/master/gnumake-4.2.1-x64.exe', 'C:\projects\google-api-nodejs-client\make.exe')
```

Comment:
It's probably not okay to remove this - it installs GNU `make`.

`make` is required for `npm test` which runs `make test ..`

Reply:
Fixed this by getting rid of the makefile and dropping the requirement on make 🙃